code/modules/genetics/stationary_scanner_console.dm

```diff
@@ -0,0 +1,303 @@
+/obj/machinery/dna_console
+	name = "Chrysalis controller"
+	desc = "A stationary computer."
+	icon = 'icons/obj/eris_genetics.dmi'
+	icon_state = "console_base"
+	density = TRUE
+	anchored = TRUE
+//	circuit = /obj/item/electronics/circuitboard/dna_console
+	use_power = IDLE_POWER_USE
+	idle_power_usage = 60
+	active_power_usage = 10000
+	req_access = list(access_research_equipment)
+	var/color_key = "yellow"
+	var/error = FALSE
+	var/obj/machinery/cryo_slab/linked
+	var/linked_direction
+	var/obj/item/computer_hardware/hard_drive/portable/usb
+
+
+/obj/machinery/dna_console/initalize_statverbs()
+	if(req_access.len)
+		add_statverb(/datum/statverb/hack_console)
+
+
+/obj/machinery/dna_console/Initialize(mapload, d=0)
+	. = ..()
+	color_key = default_dna_machinery_style
+	establish_connection()
+	update_icon()
+
+
+/obj/machinery/dna_console/New()
+	..()
+	color_key = default_dna_machinery_style
+	establish_connection()
```

Comment:
consider adding a verb for manual connections / disconnections , so people can rebuild or remove it